tests/rt.sh

```diff
@@ -129,9 +129,99 @@ readonly RT_SINGLE_CONF='rt_single.conf'
 
 source detect_machine.sh # Note: this does not set ACCNR. The "if" block below does.
 source rt_utils.sh
-
 source module-setup.sh
 
+ACCNR=${ACCNR:-""}
+if [[ -z "$ACCNR" ]]; then
+  echo "Please use -a <account> to set group account to use on HPC"
+  exit 1
+fi
+
+# Display the machine and account using the format detect_machine.sh used:
+echo "Machine: " $MACHINE_ID "    Account: " $ACCNR
```

Comment:
Having this before the while getopts will cause anyone who uses -a to fail. It needs to go directly after the while getopts to work. Why was this part moved before?

Reply:
Thanks!